<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramVisualizer</class>
 <widget class="QWidget" name="ProgramVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Program Visualization</string>
  </property>
  <widget class="QGraphicsView" name="viewer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>781</width>
     <height>381</height>
    </rect>
   </property>
   <property name="transformationAnchor">
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>1061</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="treeWidgetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Program Browser&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="textEditLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Text of Currently Selected Item&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="viewerLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Program Visualization&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="functionSelectorLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Function Displayed:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="functionSelector">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Refresh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="updateSourceFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Update the main source file to reflect changes made in the text editor</string>
   </property>
   <property name="text">
    <string>Update Source File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jumpTo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Move cursor in main editor window to the location of this item</string>
   </property>
   <property name="text">
    <string>Take Me There!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeFunction">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Make the function called in this item the currently displayed function</string>
   </property>
   <property name="text">
    <string>Display This Function</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
